### DISCUSSION TITLE
Future of the Discord server?

### DISCUSSION THREAD

**Discussion by HalogenGam3r** [Inori/GPCS4#117] [General]
The discord server you hosted is kinda dying.
Some people have actually been hacked from the nitro scams, and all of that.
and you kinda aren't active on Discord. 
Do you wish to continue and improve the server?

**Comment by Inori** [comment]
Sorry for that, currently I just don't have enough time to maintain everything.
I want to focus on code.
I will be back to discord some day when I think it's the right time.
